oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_header.less

```diff
@@ -0,0 +1,327 @@
+/*******************************
+            Theme
+*******************************/
+
+/*
+  The type is the type of the component as defined in semantic-ui - element, collection, view, module
+  For simple component, 'element' is the correct type
+*/
+@type: "view";
+
+/*
+  This is the name of the component - though not required, keep it the same as the filename
+*/
+@element: "page_header";
+
+/*******************************
+
+    DO NOT FORGET !!!
+
+Edit *theme.config* in your site 
+and add (replace 'sample' with the
+name of the component from above)
+
+@sample : 'default';
+
+*******************************/
+
+@import (multiple) "../../theme.config";
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+ */
+
+.ui.header-component {
+  margin-top: 0;
+  background-color: @pageHeaderBackgroundColor;
+  border-bottom: @pageHeaderBottomBorder;
+  width: 100%;
+  color: @pageHeaderTextColor;
+  #header-search-bar {
+    width: 100%;
+    .ui.input {
+      width: 100% !important;
+    }
+  }
+  .invenio-header-container {
+    padding: @pageHeaderPaddingTopBottom @pageHeaderPaddingLeftRight;
+  }
+
+  #invenio-nav {
+    border-radius: 0;
+    min-height: @pageHeaderMinHeight;
+    .search-bar {
+      min-width: 40%;
+    }
+    .ui.icon.input {
+      width: 100%;
+    }
+  }
+
+  #invenio-menu {
+    box-shadow: none;
+    gap: 1rem;
+
+    .item.main-nav-links a {
+      color: @pageHeaderTextColor !important;
+      font-size: 1.2rem;
+    }
+    .item.main-nav-links a:hover {
+      border-radius: @borderRadius;
+      @media all and (min-width: @largestTabletScreen) {
+        background-color: @pageHeaderMainNavLinkLargeScreen !important;
+      }
+    }
+  }
+
+  #invenio-nav.ui.menu {
+    height: @relative75px;
+    position: relative;
+
+    .ui.menu {
```

Comment:
Is selector for `.ui.menu` needed here? Is there such nested menu component under `#invenio-nav`? Maybe `#invenio-menu` was meant here? I would use id selector then :)